### ISSUE TITLE
Apple IIe with aux slot empty causing MAME to crash

### ISSUE THREAD

**Issue by tomcw** [mamedev/mame#13111]
### MAME version

0.272 (mame0272)

### System information

Windows 11, 64-bit, version 24H2

### INI configuration details

```INI
I don't have any file called "mame.ini"
```

### Emulated system/software

Enhanced Apple IIe (apple2ee)

### Incorrect behaviour

MAME executable crashes!

### Expected behaviour

Shouldn't crash :-)

### Steps to reproduce

1. Start MAME with this command line:
```
   mame apple2ee -aux "" -w -sl6 ""
```
2. type: "CALL -151"
3. type: "C050 N C05E N C00D:0"

Results in the following crash:
```
-----------------------------------------------------
Exception at EIP=00007ff668854d78 (void a2_video_device::lores_update<true>(screen_device&, bitmap_ind16&, rectangle const&, int, int) [clone .isra.0]+0x02c8): ACCESS VIOLATION
While attempting to read memory at 0000000000000400
-----------------------------------------------------
RAX=0000000000000400 RBX=0000000000000000 RCX=0000000000000000 RDX=0000022c281a34f0
RSI=000000f46bef9070 RDI=0000000000000028 RBP=0000022c2294e1b0 RSP=000000f46bef8fc0
 R8=0000022c28024260  R9=0000000000000000 R10=0000000000000000 R11=0000000000000028
R12=000000f46bef9070 R13=0000000000000000 R14=0000022c28024260 R15=0000022c22963820
-----------------------------------------------------
Stack crawl:
  000000f46bef9190: 00007ff668854d78 (void a2_video_device::lores_update<true>(screen_device&, bitmap_ind16&, rectangle const&, int, int) [clone .isra.0]+0x02c8)
  000000f46bef9220: 00007ff66885cb70 (unsigned int a2_video_device::screen_update<(a2_video_device::model)1, false, false>(screen_device&, bitmap_ind16&, rectangle const&)+0x01b0)
  000000f46bef92c0: 00007ff667fadd8c (screen_device::update_partial(int)+0x021c)
  000000f46bef9340: 00007ff66b3f992a (video_manager::finish_screen_updates()+0x00ca)
  000000f46bef93d0: 00007ff66b3fb808 (video_manager::frame_update(bool)+0x01f8)
  000000f46bef9440: 00007ff667faacb3 (screen_device::vblank_begin(int)+0x0233)
  000000f46bef94b0: 00007ff667fd1dab (device_scheduler::timeslice()+0x014b)
  000000f46bef9610: 00007ff667fcef0f (running_machine::run(bool)+0x01bf)
  000000f46befec20: 00007ff66b3112fb (mame_machine_manager::execute()+0x020b)
  000000f46beff000: 00007ff66f1c546f (cli_frontend::start_execution(mame_machine_manager*, std::vector<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::allocator<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > > const&)+0x03cf)
  000000f46beff2c0: 00007ff66f1c5a7d (cli_frontend::execute(std::vector<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::allocator<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > >&)+0x004d)
  000000f46beff320: 00007ff66b30b979 (emulator_info::start_frontend(emu_options&, osd_interface&, std::vector<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::allocator<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > >&)+0x0029)
  000000f46beff6f0: 00007ff670ebdd8c (luaopen_lfs+0xf97edc)
  000000f46beff7c0: 00007ff663da13b1 (__tmainCRTStartup+0x0231)
  000000f46beff7f0: 00007ff663da14e6 (mainCRTStartup+0x0016)
  000000f46beff820: 00007ff9f60ee8d7 (BaseThreadInitThunk+0x0017)
  000000f46beff870: 00007ff9f705fbcc (RtlUserThreadStart+0x002c)

[process exited with code 3221225477 (0xc0000005)]
You can now close this terminal with Ctrl+D, or press Enter to restart.
```

### Additional details

NB. These variants don't crash:
- mame apple2ee -aux std80 -w -sl6 ""
- mame apple2ee -aux ext80 -w -sl6 ""

NB. This issue also occurs in 0.267.

**Comment by Robbbert** [comment]
Can't confirm this. There was a beep and the screen turned into coloured blobs.

**Comment by chungy** [comment]
Crashes on Linux x86_64 too.

**Comment by rb6502** [comment]
This is fixed in 36db9ef along with the same case for double hi-res.  I'm not actually sure what a real 64K IIe would do if you tried to enable DLR or DHR but I can imagine plausible cases where it's not what you'd expect.
